### ISSUE TITLE
AWS V2 Support

### ISSUE THREAD

**Issue by BelalNafar** [garystafford/jenkins-devops#3]
If you try to deploy a CloudFormation template with parameters from file like:
`
aws cloudformation deploy --stack-name X --template-file X --parameter-overrides file://./env/test.env
`
You will get error says that parameter-overrides should be a valid key pair.
Solution is to upgrade AWS CLI to v2 in Dockerfile

# Install AWS CLI V2
RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
RUN unzip awscliv2.zip
RUN ./aws/install

**Comment by garystafford** [comment]
Makes sense, you should always use the latest AWS CLI and version. Note this project has been dormant for 5 years and requires significant upgrades to work effectively.
